ci/test_integration.sh

```diff
@@ -19,6 +19,15 @@ echo "==== ASR (backend=chainer) ==="
 ./run.sh --stage 3 --backend chainer
 echo "==== ASR (backend=pytorch num-encs 2) ==="
 ./run.sh --stage 2 --train-config ./conf/train_mulenc2.yaml --decode-config ./conf/decode_mulenc2.yaml --mulenc true
+
+# test transducer recipe
+echo "=== ASR (backend=pytorch, model=rnnt) ==="
+./run.sh --stage 2 --train-config conf/train_transducer.yaml \
+         --decode-config conf/decode_transducer.yaml
+echo "=== ASR (backend=pytorch, model=transformer-transducer) ==="
+./run.sh --stage 2 --train-config conf/train_transformer_transducer.yaml \
+         --decode-config conf/decode_transducer.yaml
+
```

Comment:
Good good good. This makes Transducer live forever. I think the can start from `--stage 3` because previous tests performed the data preparation.

Reply:
I remember now why I started from stage 2, it is because transducer test is after `./run.sh --stage 2 --train-config ./conf/train_mulenc2.yaml --decode-config ./conf/decode_mulenc2.yaml --mulenc true` which would throw the following error if I started from stage 3 : 

```
...
File "/home/fboye/espnet/espnet/asr/pytorch_backend/asr.py", line 504, in <lambda>
    dataset=TransformDataset(train, lambda data: converter([load_tr(data)])),
  File "/home/fboye/espnet/espnet/asr/pytorch_backend/asr.py", line 257, in __call__
    xs, ys = batch[0]
ValueError: too many values to unpack (expected 2)
```